.env.template

```diff
@@ -0,0 +1,5 @@
+# if you've got different settings than from docker
+
+VITE_DEV_URL="localhost"
+VITE_DEV_PORT="8080"
+VITE_DEV_DIRECTORY="game"
```

Comment:
These three have been renamed from when we were pairing earlier @OotTheMonk , for some security reason I have to prefix the ENV variables with `VITE` for it to work haha.